styleguide/_themes/global/scss/banners.scss

```diff
@@ -36,31 +36,125 @@ $banner-text-margin-medium: 10%;
 
 .banner-headline {
   @include heading($white);
-  font-size: 3em;
-  line-height: 1em;
+  font-size: 4rem;
+  line-height: 3.75rem;
   text-align: center;
 }
 
 
-.imacBanner-container {
-  background-color: $brand-primary;
+.imacBanner {
+  background-image: url('https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/bananers/backgrounds/o365-overview.jpg');
+  background-size: cover;
   color: $white;
+  margin-top: 50px;
+  overflow: hidden;
 
-  &.large {
+  &.small {
+    height: 250px;
+    padding-top: 50px;
+  }
+
+  &.medium {
     height: 300px;
-    padding-top: 30px;
+    padding-top: 50px;
+  }
+
+  &.large {
+    height: 350px;
+    padding-top: 50px;
   }
 }
 
+.imacBanner-container {
+  @include make-container;
+}
+
+.imacBanner-textRow {
+  @include make-row;
+}
+
 .imacBanner-headline {
   @include heading($white);
+  font-family: 'Khand';
   font-size: 4rem;
   font-weight: 700;
-  line-height: 3.75rem;
+  line-height: .9;
```

Comment:
Or actually on second thought, `line-height: .9em;` has the exact same effect and is more consistent with what we're wanting. `em` is always relative to the closest font-size, which is being set a few lines above, so this will always be 90% of the font size.